scripts/build_example

```diff
@@ -53,6 +53,7 @@ if [[ -z "${EXAMPLE+z}" ]]; then
   exit 1
 fi
 
+mkdir -p "out/examples/${EXAMPLE}"
```

Comment:
```suggestion
mkdir --parents "out/examples/${EXAMPLE}"
```

Reply:
By the way, expanding every argument isn't always helpful &ndash; this fails on macOS.